probe/docker/container_test.go

```diff
@@ -108,12 +107,11 @@ func TestContainer(t *testing.T) {
 		want := report.MakeSets().
 			Add("docker_container_ports", report.MakeStringSet("1.2.3.4:80->80/tcp", "81/tcp")).
 			Add("docker_container_networks", nil).
-			Add("docker_container_ips", report.MakeStringSet("1.2.3.4")).
-			Add("docker_container_ips", report.MakeStringSet("5.6.7.8")).
 			Add("docker_container_ips_with_scopes", report.MakeStringSet(";1.2.3.4")).
 			Add("docker_container_ips_with_scopes", report.MakeStringSet(";5.6.7.8")).
 			Add("docker_container_networks", report.MakeStringSet("network1"))
-
+			// Add("docker_container_ips", report.MakeStringSet("1.2.3.4")).
+			// Add("docker_container_ips", report.MakeStringSet("5.6.7.8")).
```

Comment:
I think these commented lines could be removed now given https://github.com/weaveworks/scope/pull/3591/files#diff-51f61e163768377668ac60822adf1f67L229.